companies/named-company.json

```
@@ -0,0 +1,11 @@
+{
+    "slug": "named-company",
+    "address": "NamedCompany\nStraße 5\n 1324 Stadt",
+    "name": "NamedCompany",
+    "relevant-countries": [
+        "all"
+    ],
+    "sources": [
+        "https://github.com/malte-testdev/test_repo/pull/44"
+    ]
```

Beep Boop I am a bot and this is my suggestion:
```suggestion
{
    "address": "Straße 5\n 1324 Stadt",
    "name": "NamedCompany",
    "relevant-countries": [
        "all"
    ],
    "sources": [
        "https://github.com/malte-testdev/test_repo/pull/44"
    ]
}
```

---

atm the bot's diff explodes if a line is deleted